qiita_db/metadata_template.py

```diff
@@ -760,6 +760,13 @@ def _check_special_columns(cls, md_template, obj):
             The obj to which the metadata template belongs to. Study in case
             of SampleTemplate and RawData in case of PrepTemplate
         """
+        # Check disallowed col names
+        disallowed = {'study_id', 'processed_data_id'}
```

Comment:
Would you mind creating an issue about this?

On (Mar-17-15|19:52), Joshua Shorenstein wrote:
> > @@ -760,6 +760,13 @@ def _check_special_columns(cls, md_template, obj):
> >              The obj to which the metadata template belongs to. Study in case
> >              of SampleTemplate and RawData in case of PrepTemplate
> >          """
> > +        # Check disallowed col names
> > +        disallowed = {'study_id', 'processed_data_id'}
> 
> Actually, this is already violated on the metadata_template.py file, so we need to change that ASAP unless we are droppng that directive and creating circular import fun again.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/qiita/pull/992/files#r26634923